packages/clay-css/src/scss/variables/_panels.scss

```diff
@@ -159,23 +159,48 @@ $panel-group-flush-panel-unstyled: map-deep-merge(
 
 // Panel Variants
 
-$panel-secondary-header-link: () !default;
+$panel-large-header-link: () !default;
 
-$panel-secondary: () !default;
-$panel-secondary: map-deep-merge(
```

Comment:
@p2kmgcl I know we need to change this here by design decision but this is something that has been in Clay for so long that we don't know who is using it here, so I think the best way to handle the case of `panel- secondary` is making an alias for `panel-small` the same for the React Component to ensure that whoever has `panel-secondary` still works but under the hood it will be `panel-small` in the next major release, we will be confident that we can remove this completely. Does it make sense to you?

Reply:
Yes it does :smile:

Reply:
@p2kmgcl This is too breaking imo. I think it's better if we keep `panel-large` and `panel-small` as separate modifiers that we can use on `panel` or `panel-secondary`. Those variants generally modify colors while `sm`, `md`, `lg` modify sizing such as padding, margins, font-size. We should also rename `panel-large` and `panel-small` to `panel-lg` and `panel-sm` since this is the naming convention used in Clay.

Reply:
I agree that this breaks existing behavior, but using two different variation types for colors and sizes will create variants that we do not need. If I understood it well you're suggesting:

- `displayType: secondary | unstyled`
- `size: lg | md | sm`

but with that options, we could, for example, do this:

`<ClayPanel size="lg" displayType="secondary" />`, which we do not need at all.

Also, we do not have defined styles for `md`, we would need some extra design work for that.

Reply:
Thanks @ethib137, I'll wait for Emiliano then :smile: